src/main/java/uk/gov/hmcts/reform/sscs/services/mapper/CaseDataBuilder.java

```diff
@@ -39,76 +37,66 @@
 
 @Service
 @Slf4j
-public class CaseDataBuilder {
+class CaseDataBuilder {
 
-    public static final String YES = "Yes";
-    public static final String NO = "No";
-    public static final String Y = "Y";
+    private static final String YES = "Yes";
+    static final String NO = "No";
+    private static final String Y = "Y";
```

Comment:
Again meaningful variable name should be used instead of Y

Reply:
done